packages/ui/src/utils/logo.ts

```diff
@@ -16,3 +16,24 @@ export const getLogoUrl = ({ theme, logoUrl, darkLogoUrl, isApple }: GetLogoUrl)
 
   return logoUrl;
 };
+
+type GetBrandingLogoUrl = Omit<GetLogoUrl, 'logoUrl'> & {
+  logoUrl?: Nullable<string>;
+};
+
+export const getBrandingLogoUrl = ({
```

Comment:
Chould merge with the existing getLogtoUrl method, one single `getBrandingLogtoUrl` should be fine. P.S. do you mind add a UT test for this, thx.